docs/tutorials/mastering.md

```diff
@@ -76,9 +695,68 @@ sudo microk8s.config > control-plane-cluster.yaml
 scp control-plane-cluster.yaml juju-controller.mgmt:
 ```
 
-### SD-Core User Plane
+Set alias for kubectl:
+```console
+cat << EOF | sudo tee -a ~/.bashrc
+alias kubectl="microk8s.kubectl"
```

Comment:
Do this once after installing microk8s instead of adding it to .bashrc
`sudo snap alias microk8s.kubectl kubectl`

Reply:
This is a small facility to run the kubectl commands. I thought the possibility of opening new sessions.  Maybe we can remove it.

Reply:
Done